flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +65,59 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+class FakeCinderClient(object):
+    """"
+    Mock implementation of the cinder volume manager to use in the Cinder
+    Destroy tests.
+    This class will actually do nothing as we want to emulate a non-responsive
+    cinder volume.
+    """
+    def delete(self, volume_id):
+        """
+        A no-op delete.
+        We do not need the delete to do anything for the timeout tests.
+        """
+
+    def get(self, volume_id):
+        """
+        A no-op get.
+        Right now, we don't need a full mock of the get, we just need the
+        get not to raise any exception so the destroy volume timeout test
+        actually times out - it will be trying to get the volume in a loop
+        after deleting it, expecting a ``CinderNotFound`` exception that will
+        never happen, so we can verify that the timeout exception is rised
+        (see FLOC-1853)
+        """
+
+
+class CinderDestroyTests(SynchronousTestCase):
+    """
+    Tests for destroy_volume.
+    """
+
+    # Test added with the fix of the issue FLOC-1853
+    def test_timeout(self):
+        """
+        ``CinderBlockDeviceAPI.destroy_volume`` raises
+        ``TimeoutException`` if the volume is not removed within
+        some time (``_currentTimeout`` will define how long it will wait).
```

Comment:
``_currentTimeout`` isn't the actual name of the attribute you're setting. And maybe rely on test body to explain this?